src/running-a-sora-testnet-node.md

```diff
@@ -0,0 +1,299 @@
+
+#### 3.2. Configure Port Access
+
+Ensure that port `1337` is open on your firewall, or add the necessary rules in your provider’s security group settings to allow inbound traffic.
+
+#### 3.3. Specify Your Keys in the Docker Compose Configuration
+
+Edit the previously downloaded `docker-compose.volunteer.yml` file as follows:
+
+```yml
+# For the attached client
+ACCOUNT_PUBLIC_KEY: <your_public_key>
+ACCOUNT_PRIVATE_KEY: <your_private_key>
+# For the peer
+PUBLIC_KEY: <your_public_key>
+PRIVATE_KEY: <your_private_key>
+P2P_PUBLIC_ADDRESS: <your_advertised_host>:1337
+```
+
+#### 3.4. Start the Docker Container
+
+From the folder containing the `docker-compose.volunteer.yml` file run the following command to launch your node:
+
+```bash
+docker compose -f docker-compose.volunteer.yml up -d
+```
+
+**Example output:**
+
+```log
+[+] Running 2/2>docker compose -f docker-compose.volunteer.yml up -d
+ ✔ Network downloads_default     Created                             0.1s
+ ✔ Container downloads-irohad-1  Started
+```
+
+### 4. Check Node Status
+
+Once your the administrators register your node, verify its status using one of these commands:
+
+```bash
+curl <your_host>:8080/status
+curl <your_host>:8080/peers
+```
+
+::: tip Note
+
+If the peer list is empty, your node may not be registered, or there might be network issues. Troubleshoot with testnet admins.
+
+:::
+
+## Perform Transactions via Your Node
+
+### 1. Send and Inspect a Mock Transaction
+
+To listen for incoming transactions, attach a shell to the running container:
+
+```bash
+docker exec -it downloads-irohad-1 bash
+```
+
+And once you're in the container's shell, run:
+
+```bash
+cd /config
+iroha events transaction
+```
+
+This will create a transaction listener.
+
+**Example output:**
+
+```bash
+C:\Users\user>docker exec -it downloads-irohad-1 bash
+iroha@263a8e4bbbd1:/$ cd /config
+iroha@263a8e4bbbd1:/config$ iroha events transaction
+Listening to events with filter: Pipeline(Transaction(TransactionEventFilter { hash: None, block_height: None, status: None }))
+```
+
+In another instance of an attached shell, send a mock transaction:
+
+```bash
+cd /config
+iroha transaction ping --msg "This is an extraordinary mock transaction"
+```
+
+**Example output:**
+
+```json
+"23EC79207A5573333057A4836533A72ED015AADE4DABC00CA8676120C919DE67"
+```
+
+::: tip Note
+
+If the account is not found, your account may not be registered. Troubleshoot with testnet admins.
+
+:::
+
+If the transaction listener is running, you should see a report of the approved transaction:
+
+```json
+{
+  "Pipeline": {
+    "Transaction": {
+      "hash": "23EC79207A5573333057A4836533A72ED015AADE4DABC00CA8676120C919DE67",
+      "block_height": 2,
+      "status": "Approved"
+    }
+  }
+}
+```
+
+### 2. Query Transaction Details
+
+Retrieve details of a specific transaction using its hash:
```

Comment:
```suggestion
Retrieve the details of a specific transaction using its hash:
```

Reply:
Resolved in commit 88c29302a00fa18434ec2ba84d3c872148245122